mani_skill/examples/motionplanning/panda_stick/solutions/draw_triangle.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import sapien
+from mani_skill.envs.tasks import PushCubeEnv
+from mani_skill.examples.motionplanning.panda_stick.motionplanner import \
+    PandaStickMotionPlanningSolver
+from PIL import Image
+
+
+def solve(env: PushCubeEnv, seed=None, debug=False, vis=False):
+    env.reset(seed=seed)
+    planner = PandaStickMotionPlanningSolver(
+        env,
+        debug=debug,
+        vis=vis,
+        base_pose=env.unwrapped.agent.robot.pose,
+        visualize_target_grasp_pose=vis,
+        print_env_info=False,
```

Comment:
can you lower the max allowed joint velocities? Right now the result drawings are a bit too fast (dots are too spread out).

maybe lets tune it so that the typical solve time is about the 200-250 steps mark